pages/video-download/video-download.view.html

```diff
@@ -1,11 +1,26 @@
-<div ng-controller="VideoDownloadController as VideoDownload">
-	YEEEEEPALE!! {{VideoDownload.id}}
-
-	<div>
-		{{VideoDownload.video.data.title}}
-		{{VideoDownload.video.data.description}}
-	</div>
-
-	<input type="text" ng-model="VideoDownload.code">
-	<a ng-click="VideoDownload.download()" target="_blank">Download</a>
+<!--YEEEEEPALE!! {{VideoDownload.id}}-->
+<div ng-controller="VideoDownloadController as VideoDownload" class="download-page">
+	<article>
+		<video poster="../../img/video1.jpg" class="video-download-page"></video>
```

Comment:
`poster="../../img/video1.jpg"` Shall better be the `{{VideoDownload.video.data.poster}}`, that is the actual video poster.

Also, just to make sure, have we talked with Juan about default posters? No worry so much now about it, is just to know what's the actual status ;)